azure-iot-common/tests/conftest.py

```diff
@@ -0,0 +1,8 @@
+import sys
+
+collect_ignore = []
```

Comment:
>collect_ignore [](start = 0, length = 14)

nice!  I didn't know about this feature.  I might have done some things differently in the past had I known this.  Thanks!